xml/System/ArgumentOutOfRangeException.xml

```diff
@@ -132,9 +132,7 @@
      [!code-csharp[System.ArgumentOutOfRangeException#18](~/samples/snippets/csharp/VS_Snippets_CLR_System/System.ArgumentOutOfRangeException/cs/NoFind2.cs#18)]
      [!code-vb[System.ArgumentOutOfRangeException#18](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/System.ArgumentOutOfRangeException/vb/NoFind2.vb#18)]  
   
--  
-  
- You've attempted to extract a substring that is outside the range of the current string.  
+-   You've attempted to extract a substring that is outside the range of the current string.  
```

Comment:
The formatting (levels of indentation) in this topic is a complete disaster. I had started to make a list of items that you might easily fix if you wanted, but it's too extensive. I'll open an issue to fix it, and once this is merged, I'll address this issue.

Reply:
Sounds good @rpetrusha. I've removed one file from this set of changes that needs to be merged after we switch the engine to markdig, otherwise, it breaks what we have right now.